src/Loading/Plugins/babylon.babylonFileLoader.ts

```diff
@@ -649,7 +649,8 @@ module BABYLON {
                     if (parsedData.createDefaultSkybox === true) {
                         var skyboxScale = (scene.activeCamera !== undefined && scene.activeCamera !== null) ? (scene.activeCamera.maxZ - scene.activeCamera.minZ) / 2 : 1000;
                         var skyboxBlurLevel = parsedData.skyboxBlurLevel || 0;
-                        scene.createDefaultSkybox(undefined, true, skyboxScale, skyboxBlurLevel);
+                        var skyboxIsPBR = parsedData.skyboxIsPBR;

```

Comment:
```suggestion
                        var skyboxIsPBR = parsedData.skyboxIsPBR !== undefined ? arsedData.skyboxIsPBR : true;
                        scene.createDefaultSkybox(undefined, skyboxIsPBR, skyboxScale, skyboxBlurLevel);
```
undefined is equivalent to scene.environmentTexture